test/geodesy_test.dart

```diff
@@ -344,4 +344,71 @@ void main() {
           closeTo(expectedPoints[i].longitude, tolerance));
     }
   });
+
+  test('Calculate the area of a square', () {
+    // Create a square with four vertices
+    List<LatLng> squarePoints = [
+      const LatLng(0.0, 0.0),
+      const LatLng(0.0, 1.0),
+      const LatLng(1.0, 1.0),
+      const LatLng(1.0, 0.0),
+    ];
+
+    // Calculate the area of the square
+    double area = geodesy.calculatePolygonArea(squarePoints);
+
+    // The area of a square with side length 1 is 1 square meter
+    expect(area, equals(1.0));
+  });
+
+  test('Calculate the length of a straight line in meters', () {
+    // Create a straight line with two points
+    List<LatLng> linePoints = [
+      const LatLng(0.0, 0.0),
+      const LatLng(0.0, 1.0),
+    ];
+
+    // Calculate the length of the line in meters
+    double length = geodesy.calculatePolyLineLength(linePoints);
+
+    // The length of a straight line from (0,0) to (0,1) in meters
+    // is 111319.0 meters
+    expect(length, equals(111319.0));
+  });
+
+  ///
```

Comment:
can be deleted?

Reply:
It can not be deleted because this length is in milli meters.. And In Killometer it should be 0.1...